console-app-dotnetcore/Program.cs

```diff
@@ -10,6 +12,8 @@ class Program
 
         static void Main(string[] args)
         {
+            LogManager.Adapter = new Common.Logging.Simple.DebugLoggerFactoryAdapter();
```

Comment:
Just making it so you can see the log output of the console app - at least, if you're running in the IDE (it will appear in the Application Output panel).